client/me/purchases/cancel-purchase/button.jsx

```diff
@@ -97,8 +97,8 @@ class CancelPurchaseButton extends Component {
 			if ( success ) {
 				this.props.successNotice(
 					translate(
-						'%(purchaseName)s was successfully cancelled. It will be available ' +
-							'for use until it expires on %(subscriptionEndDate)s.',
+						'Auto-renew for %(purchaseName)s was successfully disabled. Your paid features will be available ' +
+							'for use until your plan expires on %(subscriptionEndDate)s.',
```

Comment:
This text is used for more than just plans (for example, it also appears when cancelling a domain) so if you're adding any plan-specific language in here you'd have to make sure it only appears for plans.